eng/pipelines/templates/jobs/ci.yml

```diff
@@ -68,6 +68,28 @@ jobs:
         displayName: 'Use Python 3.6'
         inputs:
           versionSpec: '3.6'
+      - pwsh: |
+          New-Item -Path "$(Build.sourcesdirectory)" -Name "changedFiles" -ItemType "directory"
+          git diff $targetBranch HEAD --name-only | ForEach-Object {Copy-Item -Path $_ -Destination "$(Build.sourcesdirectory)/changedFiles"}
```

Comment:
FYI- I think we are at a stage where we should have a common helper to find the list of changed files as this approach works sometimes it doesn't always.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/2613